include/ccf/json_handler.h

```diff
@@ -261,6 +261,9 @@ namespace ccf
   inline jsonhandler::JsonAdapterResponse make_error(
     http_status status, const std::string& code, const std::string& msg)
   {
+    // Debug-only log for BFT, which may swallow frontend error results.
```

Comment:
This comment does not match the implementation, maybe make keep this both CFT and BFT but i am not sure everyone would agree on this

Reply:
In CFT, this results in an error returned to the client, which is why we don't typically log it. In BFT of course, most execution doesn't yield anything to the client, and so this become useful for debugging.

What happens at least once per request would typically be TRACE rather than DEBUG, but I have no objection to adding this otherwise in all cases. We know we have too much logging and we need to look into named loggers.

Reply:
Yeah "for BFT" means "I'm adding this because of BFT" :-)